index.html

```diff
@@ -0,0 +1,127 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      rel="stylesheet"
+      href="https://sad-haibt-42112c.netlify.app/styles/main.css"
+    />
+    <link rel="stylesheet" href="/styles/wishlist.css" />
+    <title>Your Wishlist | Sanguine Store</title>
+  </head>
+  <body>
+    <nav class="navigation">
+      <div class="nav-logo">SANGUINE store</div>
+      <label for="search" class="text-size-sm">
+        <input
+          type="search"
+          class="search-box padding-md border-rad-md"
+          name="search"
+          id="search"
+          placeholder="What are you looking for?"
+        />
+        <a href="" class="options text-dec-none">Search</a>
+      </label>
+      <ul class="list-style-none list-inline">
+        <li>
+          <a class="options link" href="">profile</a>
+        </li>
+        <li>
+          <a class="options link link-active" href="">wishlist</a>
+        </li>
+        <li>
+          <a class="options link" href="">cart</a>
+        </li>
```

Comment:
Suggestion: To wrap the <li> tag with the <a>. It depends on use case like if the <li> has some padding then even if the user clicks on the fill area the <a> tag gets triggered otherwise it would only work when clicked on the text. you can opt according to your personal preference.

Reply:
Sorry but I couldn't understand, do you mean not having the `<a>` wrapped inside `<li>`?

Reply:
Yes Parvez, I have updated the comment according to markdown now. My bad for previous one 😅